installer/conf/omsagent.d/patch_management_immediate_run.conf

```diff
@@ -0,0 +1,12 @@
+<source>
+  type exec
+  tag oms.patch_management_immediate_run
+  command /opt/microsoft/omsconfig/Scripts/PerformInventory.py --InMOF /etc/opt/microsoft/omsagent/conf/omsagent.d/patch_management_inventory.mof --OutXML /etc/opt/omi/conf/omsconfig/configuration/CompletePackageInventory.xml > /dev/null && cat /etc/opt/omi/conf/omsconfig/configuration/CompletePackageInventory.xml
```

Comment:
Yes, as out run_interval is now set to 3 hours (earlier, it was 12 hours), the users do not see any data as soon as they onboard. From fluentd forums, the preferred way was to crontab the python separately and use in_tail (https://groups.google.com/forum/#!topic/fluentd/Yq3BRYxYQuI), but then we lose the flexibility to restart if anything happens to the python script as well as the installers then woudl be changed, which was not feasible.

The data is the same from oms.patch_management_immediate_run and oms.patch_management. Also, the oms.patch_management_immediate_run should run only once based on the fluentd code.